### ISSUE TITLE
[12.0][BUG] agreement_legal: company's primary contact selection

### ISSUE THREAD

**Issue by ygol** [OCA/contract#343]
1. edit an agreement.
2. select a primary contact for the company
```
Error:
Error: AttributeError: object has no attribute 'partner_id'

http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:205
Traceback:
__getattribute__@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:205:7
py.PY_getAttr@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:168:176
py.evaluate@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:272:11
py.evaluate@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:267:111
py.evaluate@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:269:99
py.eval@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:274:284
eval_domains/<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:413:136
_.forEach@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:12:558
_.mixin/</_.prototype[name]@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:69:526
eval_domains@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:412:265
pyeval@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:428:24
stringToArray@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:383:2283
_getDomain@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:1438:167
_search@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:1209:593
source/<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:1202:224
_.forEach@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:12:558
source@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:1202:55
_search@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1716:147
$.widget/</proxiedPrototype[prop]</<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1276:334
search@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1716:13
$.widget/</proxiedPrototype[prop]</<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1276:334
$.widget.bridge/$.fn[name]/<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1281:31
each@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:625:767
each@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:613:643
$.widget.bridge/$.fn[name]@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:1278:253
_onInputClick@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/563-5dd5e44/web.assets_backend.js:1214:729
proxy/<@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:3643:11
dispatch@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:892:451
add/elemData.handle@http://3373406-12-0-577fa0.runbot3.odoo-community.org/web/content/557-da04fe9/web.assets_common.js:865:173
```

**Comment by ygol** [comment]
same problem for field **"notification_address_id"**

**Comment by pedrobaeza** [comment]
Fixed with #473
